docs/reference/commandline/dockerd.md

```diff
@@ -977,3 +977,55 @@ has been provided in flags and `cluster-advertise` not, `cluster-advertise`
 can be added in the configuration file without accompanied by `--cluster-store`
 Configuration reload will log a warning message if it detects a change in
 previously configured cluster configurations.
+
+
+## Running multiple daemons
+
+This section describes how to run multiple Docker daemons on a single host. To
+run multiple daemons, you must configure each daemon so that it does not
+conflict with other daemons on the same host. You can set these options either
+by providing them as flags, or by using a [daemon configuration file](#daemon-configuration-file).
+
```

Comment:
Given the discussion, I think we may want to add a note/warning here that running multiple daemons on a single host is considered "experimental". There's still issues people can run into that we may not cover in this description. (for example, as mentioned, IPTables, plugins) also not entirely sure if multiple daemons talking to the same `containerd` instance is asking for trouble (thinking of the "restart daemon without restarting containers" feature being worked on for docker 1.12).